torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5900,6 +5906,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                        SkipInfo('TestUnaryUfuncs', 'test_out_arg_all_dtypes'),
                    )),
     OpInfo('roll',
+           ref=np.roll,
```

Comment:
roll seems like a good candidate to start with, since it doesn't seem to have type promotion issues or other oddities that require writing a more involved reference

Reply:
Agreed!